hraven-etl/src/test/java/com/twitter/hraven/etl/TestJobHistoryFileParserHadoop2.java

```diff
@@ -90,7 +76,7 @@ public void testCreateJobHistoryFileParserCorrectCreation() throws IOException {
             JobHistoryKeys.JOB_STATUS.toString().toLowerCase()));
     
     assertNotNull(statusRecord);
-    assertEquals((String)statusRecord, JobHistoryFileParserHadoop2.JOB_STATUS_SUCCEEDED);
+    assertEquals((String)statusRecord, JobHistoryFileParserHadoop2.JOB_STATUS_SUCCESS);
```

Comment:
how was the test passing earlier? did you change the history file in src/test/resources to match the new STATUS record name?